__tests__/TimelineTest.js

```diff
@@ -15,7 +15,7 @@ describe('Timeline', () => {
 							 "profileImageUrl":"http://pbs.twimg.com/profile_images/1031635661701308416/C0nXsZv0_normal.jpg",
 							 "statusId":"1035247174618099712",
 							 "createdAt":1535657135000}];
-
```

Comment:
This is a good time to start being mindful of whitespace changes as a courtesy to people reviewing your code